scripts/job.pbs

```diff
@@ -80,7 +71,23 @@ teardown()
 	log_echo "  $CHECKPOINT_FILE"
 	cp -p "$TEMP_DIR/checkpoint.chk" "$CHECKPOINT_FILE"
 
-	log_echo "Teardown complete, job done ;)"
+	log_echo "Teardown complete"
+}
+
+fchk()
+{
+	log_echo "Formatting checkpoint file:"
+	log_echo "  $FORMATTED_FILE"
+	
+	# creates directory for fchk files
+	mkdir -p $PBS_O_WORKDIR/fchkpoints/"
+	formchk "$CHECKPOINT_FILE" "$FORMATTED_FILE"
```

Comment:
I tried this and it wasn't working. I'll check it again. Thought maybe it was because it was trying to format halfway finished chkfiles